docs/translations.md

```diff
@@ -1,43 +1,777 @@
+`X-Client-Region` header based on the visitor's IP address. There is no way for a user to
+override this region selection. A tester can use a [VPN][] to change their IP address
+and their detected region.
+
+The `X-Client-Region` header is sometimes missing. In stage and production, some IP
+addresses do not have a clear region. The development deployment does not have the
+header. Local development also omits the header.
+
+Without a `X-Client-Region` header, the code guesses a region from the `Accept-Language`
+header. The language fallback looks for a language code with a regional variant. For
+example, the language code `"es-MX"` (Mexican Spanish) implies region code `"MX"`
+(Mexico). When there is no regional variant, the code guesses a probable region from a
+[lookup table][]. The lookup table uses Unicode [Common Locale Data Repository][] (CLDR)
+[Supplemental Data][]. The [Language-Territory Information][] data includes estimates of
+language speakers by region.
+
+[Common Locale Data Repository]: https://cldr.unicode.org/
+[Language-Territory Information]: https://www.unicode.org/cldr/charts/42/supplemental/language_territory_information.html
+[Supplemental Data]: https://www.unicode.org/cldr/charts/42/supplemental/index.html
+[VPN]: https://www.mozilla.org/products/vpn
+[lookup table]: https://github.com/mozilla/fx-private-relay/blob/f5c5ebf568639810db45de1ebb69f2498600d58c/privaterelay/utils.py#L72-L216
+
+### Identifying Available Plans and Prices
+
+The user's region determines what premium plans are available. The API sends this data
+in [/api/v1/runtime_data][]. The front end adjusts content based on availability in the
+user's region.
+
+The subscription platform (version 2) uses [Stripe][]. A [Stripe product][] represents
+each Relay plan, such as the premium email service. A [Stripe price][] represents how a
+user will pay. The price has an associated product, a currency, tax details, and a term.
+Many Relay plans have monthly and yearly subscription terms. The VPN bundle has yearly
+terms.
+
+The subscription platform associates a price with a region. Product reports may use this
+to segment purchasers by region.
+
+The subscription platform associates a price with a language. On the subscription page,
+product details appear in the price language. The rest of the subscription page appears
+in the user's preferred language. For most users, the price language and preferred
+language should be the same.
+
+Many regions have their own price, such as Denmark, which uses the Danish language and
+the krone. Others share the price of a nearby region, such as Austria, which uses
+Germany's prices in German and Euros. A minority are more complex, choosing a price
+based on language. Belgium users share the price in Euros of a neighbor, based on
+language. Switzerland has separate prices for German, French, and Italian speakers.
+All the Swiss prices are in Swiss Francs. See [privaterelay/plans.py][] for details.
```

Comment:
(question) From reading the previous section "Identifying the User's Region" I now understand the Relay's logic for getting regional data--region and language. From the regional data, Relay surfaces the available product and plans  which are mapped in `plans.py`. But does the subscription platform also use the same method on identifying the user's region as Relay does? My understanding is that they use something different and could lead to the subscription platform showing different language(s)/currency on the subscription page.